installer/tue-install-impl.bash

```diff
@@ -806,17 +1036,44 @@ TUE_INSTALL_STATE_DIR=$TUE_INSTALL_GENERAL_STATE_DIR/$stamp
 mkdir -p "$TUE_INSTALL_STATE_DIR"
 
 TUE_INSTALL_GIT_PULL_Q=()
+TUE_INSTALL_HG_PULL_Q=()
 
 TUE_INSTALL_SYSTEMS=
 TUE_INSTALL_PPA=
-TUE_INSTALL_PIPS=
+TUE_INSTALL_PIP2S=
 TUE_INSTALL_PIP3S=
 TUE_INSTALL_SNAPS=
 
 TUE_INSTALL_WARNINGS=
 TUE_INSTALL_INFOS=
 
+# Make sure tools used by this installer are installed
+tue-install-system-now git python-pip python3-pip
+
+tue-install-pip2-now mercurial
```

Comment:
Why not system-now? If really pip then why not pip3?

Reply:
Apt is very old, 3.7. Pip is on 5.3. But the hgcfg extension is not yet compatible with python3.

Reply:
Okay. Then this remains unchanged for now.